tutorials/scripting/gdextension/gdextension_file.rst

```diff
@@ -50,21 +50,25 @@ Here is an example of what that can look like:
 
 .. code-block:: none
 
+    ; A comment line starts with a semicolon. This line is ignored by the engine.
     [libraries]
 
-    macos.debug = "res://bin/libgdexample.macos.template_debug.framework"
-    macos.release = "res://bin/libgdexample.macos.template_release.framework"
-    windows.debug.x86_32 = "res://bin/libgdexample.windows.template_debug.x86_32.dll"
-    windows.release.x86_32 = "res://bin/libgdexample.windows.template_release.x86_32.dll"
-    windows.debug.x86_64 = "res://bin/libgdexample.windows.template_debug.x86_64.dll"
-    windows.release.x86_64 = "res://bin/libgdexample.windows.template_release.x86_64.dll"
-    linux.debug.x86_64 = "res://bin/libgdexample.linux.template_debug.x86_64.so"
-    linux.release.x86_64 = "res://bin/libgdexample.linux.template_release.x86_64.so"
-    linux.debug.arm64 = "res://bin/libgdexample.linux.template_debug.arm64.so"
-    linux.release.arm64 = "res://bin/libgdexample.linux.template_release.arm64.so"
-    linux.debug.rv64 = "res://bin/libgdexample.linux.template_debug.rv64.so"
-    linux.release.rv64 = "res://bin/libgdexample.linux.template_release.rv64.so"
-
+    macos.debug = "bin/libgdexample.macos.template_debug.framework" ; Inline comments are also allowed.
+    macos.release = "bin/libgdexample.macos.template_release.framework"
```

Comment:
In the template, we did all the paths starting with `./`, which, if I remember correctly was to fix a MacOS issue? And we also use `.dylib` in the template and not `.framework`, which I also remember discussing being to workaround some issue.

Do you folks remember the details?

Anyway, if there was a reason for those changes, we should do them in the docs as well

Reply:
Good point! Yea, I believe the macOS load issue is still around, so changing to `./` format would be good.

And yea, the change from framework to dylib also definitely makes sense.